src/components/PhoneNumberInput/index.js

```diff
@@ -32,7 +32,6 @@ class PhoneNumberInput extends Component {
   render = () =>
     <PhoneNumber placeholder='Enter mobile number'
       onChange={this.onChange}
-      value={this.props.sms.number}
```

Comment:
2 story points for this!

Reply:
Two points cause I wasn't sure if something had changed on the redux side. And in fact, there was a small change, hence spotted the regression.